wavelength-plugins/profile.js

```diff
@@ -315,7 +315,8 @@ exports.commands = {
 				profile += '&nbsp;<font color="#24678d"><b>Group:</b></font> ' + userGroup + ' ' + devCheck(username) + vipCheck(username) + '<br />';
 				profile += '&nbsp;<font color="#24678d"><b>Registered:</b></font> ' + regdate + '<br />';
 				profile += '&nbsp;<font color="#24678d"><b>' + global.currencyPlural + ':</b></font> ' + currency + '<br />';
-				profile += '&nbsp;<font color="#24678d"><strong>EXP Level:</strong></font> ' + WL.level(toId(username)) + '<br />';
+				profile += '&nbsp;<font color="#24678d"><strong>Faction:</strong></font> ' + WL.level(toId(username)) + '<br />';
+				profile += '&nbsp;<font color="#24678d"><strong>EXP Level:</strong></font> ' + WL.getFaction(toId(username)) + '<br />';
```

Comment:
level is faction, and faction is level

lol

Reply:
*facepalm*